File: javascript-client/src/model/ReporterLoation.js
// require files in Node.js environment

if (typeof module === 'object' && module.exports) {
  
}




//export module
if ( typeof define === "function" && define.amd ) {
  define('ReporterLoation', ['jquery'],
    function($) {
      return ReporterLoation;
   });
}


var ReporterLoation = function ReporterLoation() { 
  var self = this;
  
  /**
   * datatype: String
   **/
  self.ipAddress = null;
  
  /**
   * datatype: Number
   **/
  self.latitude = null;
  
  /**
   * datatype: Number
   **/
  self.longitude = null;
  
  
  self.constructFromObject = function(data) {
    if (!data) {
      return;
    }
    
    self.ipAddress = data.IpAddress;
    
    self.latitude = data.Latitude;
    
    self.longitude = data.Longitude;
    
  }

  
  /**
   * @return {String}
   **/
  self.getIpAddress = function() {
    return self.ipAddress;
  }

  /**
   * @param {String} ipAddress
   **/
  self.setIpAddress = function (ipAddress) {
    self.ipAddress = ipAddress;
  }
  
  /**
   * @return {Number}
   **/
  self.getLatitude = function() {
    return self.latitude;
  }

  /**
   * @param {Number} latitude
   **/
  self.setLatitude = function (latitude) {
    self.latitude = latitude;
  }
  
  /**
   * @return {Number}
   **/
  self.getLongitude = function() {
    return self.longitude;
  }

  /**
   * @param {Number} longitude
   **/
  self.setLongitude = function (longitude) {
    self.longitude = longitude;
  }
  

  self.toJson = function () {
    return JSON.stringify(self);
  }
}

if (typeof module === 'object' && module.exports) {
  module.exports = ReporterLoation;
}

File: android-client/src/main/java/io/swagger/client/model/ReporterLoation.java
package io.swagger.client.model;


import io.swagger.annotations.*;
import com.google.gson.annotations.SerializedName;


@ApiModel(description = "")
public class ReporterLoation  {
  
  @SerializedName("IpAddress")
  private String ipAddress = null;
  @SerializedName("Latitude")
  private Double latitude = null;
  @SerializedName("Longitude")
  private Double longitude = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  public String getIpAddress() {
    return ipAddress;
  }
  public void setIpAddress(String ipAddress) {
    this.ipAddress = ipAddress;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public Double getLatitude() {
    return latitude;
  }
  public void setLatitude(Double latitude) {
    this.latitude = latitude;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  public Double getLongitude() {
    return longitude;
  }
  public void setLongitude(Double longitude) {
    this.longitude = longitude;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ReporterLoation {\n");
    
    sb.append("  ipAddress: ").append(ipAddress).append("\n");
    sb.append("  latitude: ").append(latitude).append("\n");
    sb.append("  longitude: ").append(longitude).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
